public/gameInfo.php

```diff
@@ -87,6 +87,8 @@
             $gate = true;
         }
     }
+} elseif (str_contains($gameTitle, '[Theme - Mature]')) {
```

Comment:
you need to honor the `v` parameter, or a user who hasn't set their profile permission can never access the page.

Reply:
Thanks, addressed in https://github.com/RetroAchievements/RAWeb/pull/1663/commits/c2190473a6afc0e5fc6ccacd1d7a6f3a524f9b9d.